### PR TITLE
Update ssvm

### PR DESCRIPTION
**Pull request by bentaculum**

### CODE REVIEW

File: tests/test_structsvm.py [hunk 1]
```diff
@@ -77,11 +80,106 @@ def test_structsvm():
...
+    solver.add_costs(
+        EdgeSelection(
+            weight=np.random.randint(-10, 10, 1)[0],
+            constant=np.random.randint(-10, 10, 1)[0],
+            attribute="noise",
+        ),
+        name="EdgeNoise",
+    )
+
+    print("====== Initial Weights ======")
```
Comment (tlambert03) [inline tests/test_structsvm.py:110]: how would you feel about removing the print statements here?  They're certainly useful for debugging, but they make it rather hard to run general `pytest -s` and make sense of the output.  perhaps an example file would be a better place to put all the prints?

Comment (bentaculum) [reply tests/test_structsvm.py]: Good point, I use `logging` in debug mode now: [70d0ea6](https://github.com/funkelab/motile/pull/23/commits/70d0ea68bef7a42c54dfe151f9bea865ff3e2c46)

Comment (bentaculum) [reply tests/test_structsvm.py]: The ilp solver in the back still talks for `pytest -s`, this needs to be addressed in `ilpy` with some verbosity flag.

**Comment by funkey** [comment]
Python 3.9 fails because of different handling of type annotations (@tlambert03, maybe you can help?).

I'll merge anyway, because the other versions pass and this PR fixes other tests.

**Comment by tlambert03** [comment]
yep!  on it now... will update #28 and ping you

**Pull request merged**